Python/import.c

```diff
@@ -492,6 +497,7 @@ PyImport_Cleanup(void)
                     PyErr_WriteUnraisable(NULL);
                     continue;
                 }
+                assert(!PyErr_Occurred());
```

Comment:
`PyObject_GetIter()`, `PyIter_Next()` and `PyObject_GetItem()` can raise and exception if corresponding slots are implemented incorrectly in extensions. `assert` looks good here.